### PR TITLE
workflow changes

### PR DESCRIPTION
**Pull request by ags1773**
To Do after merge: make `@circleci-checks/perf_lighthouse_test` as `required`, remove stg

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,11 +117,31 @@ workflows:
           context: quay-login
           requires:
             - deploy_staging
+  perf_lighthouse_test:
```
Comment (ags1773) [inline .circleci/config.yml:120]: created this workflow so that we can check if it passed/failed and allow/disallow merge into master

**Review by Deokumarjnu — APPROVED** [review]
can you remove the required from the staging lighthouse test and just make perf as mandatory ?

**Review by Athira001 — APPROVED** [review]

**Comment by ags1773** [comment]
@Deokumarjnu that we need to do from github once this gets merged to master

**Comment by ags1773** [comment]
https://github.com/quintype/ace-planning/issues/547

**Pull request merged**